### ISSUE TITLE
Allow processing SEMI-APPROVED concepts during ETL testing

### ISSUE THREAD

**Issue by lbertelo01** [RADar-AZDelta/Rabbit-in-a-Blender#22]
If you want to test your ETL in your dev branch, it would be helpful that semi-approved concepts get processed as well. Then you will have confirmation that your ETL runs errorless and is ready to be reviewed by a colleague.

**Comment by pjlammertyn** [comment]
Solved in version 0.0.36

you will need to add the --process-semi-approved-mappings CLI argument

```bash
# uninstall the package
pip uninstall -y Rabbit-in-a-Blender
# wait a moment
sleep 10
#pip install -i https://test.pypi.org/simple/ Rabbit-in-a-Blender # install from test repo
pip install --no-cache-dir Rabbit-in-a-Blender
```

**Comment by kdenturc** [comment]
while trying a lot of riab cycles to attempt a labo filled measurement table, 
command: riab --run-etl . --table measurement --process-semi-approved-mappings
ETL ran, I have the semi-approved mappings in my work tables, but not in my measurement omop table 
![image](https://github.com/RADar-AZDelta/Rabbit-in-a-Blender/assets/98580512/0558c8ad-9310-4bf0-8936-4059049584ff)
![image](https://github.com/RADar-AZDelta/Rabbit-in-a-Blender/assets/98580512/3a8cf12e-5f0a-4c8e-ac0b-ebead2c13024)
howcome?

**Comment by pjlammertyn** [comment]
solved in RiaB version 0.0.37
